lib/daru/view/adapters/highcharts/display.rb

```diff
@@ -61,7 +80,28 @@ def show_in_iruby(placeholder=random_canvas_id)
     def to_html_iruby(placeholder=random_canvas_id)
       # TODO : placeholder pass, in plot#div
       chart_hash_must_be_present
-      high_chart_iruby(extract_chart_class, placeholder, self)
+      script = high_chart_css(placeholder)
+      script << high_chart_iruby(extract_chart_class, placeholder, self)
+      script
+    end
+
+    # @param placeholder [String] ID of the div in which highchart has to
+    #   rendered
+    # @return [String] css code of the chart
+    def high_chart_css(placeholder)
+      # contains the css provided by the user as a String array
+      css_data = options[:css].nil? ? '' : options.delete(:css)
+      script = ''
+      if css_data != ''
+        script << '<style>'
+        # Applying the css to chart div
+        css_data.each do |css|
```

Comment:
Please check alignment and `script` must be `css_script` or specific name for css.